sdk/c/common.mk

```diff
@@ -0,0 +1,139 @@
+include(CMakeParseArguments)
```

Comment:
Is there any necessary to duplicate this file since we already have one in `cmake/module`.